<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>8-Puzzle</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>assets/icon.png</normaloff>assets/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QLabel" name="label_title">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>8-Puzzle Game</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_moves">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Moves: 0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>50</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>50</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="spot_1">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	border: 1px solid black;
	background-color: rgba(255, 255, 255, 5);
}

QPushButton:hover {
	background-color: rgba(255, 255, 255, 150);
}

QPushButton:pressed {
	background-color: rgba(255, 255, 255, 200);
}</string>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="spot_2">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	border: 1px solid black;
	background-color: rgba(255, 255, 255, 5);
}

QPushButton:hover {
	background-color: rgba(255, 255, 255, 150);
}

QPushButton:pressed {
	background-color: rgba(255, 255, 255, 200);
}</string>
           </property>
           <property name="text">
            <string>6</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="spot_3">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	border: 1px solid black;
	background-color: rgba(255, 255, 255, 5);
}

QPushButton:hover {
	background-color: rgba(255, 255, 255, 150);
}

QPushButton:pressed {
	background-color: rgba(255, 255, 255, 200);
}</string>
           </property>
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="spot_4">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	border: 1px solid black;
	background-color: rgba(255, 255, 255, 5);
}

QPushButton:hover {
	background-color: rgba(255, 255, 255, 150);
}

QPushButton:pressed {
	background-color: rgba(255, 255, 255, 200);
}</string>
           </property>
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="spot_5">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	border: 1px solid black;
	background-color: rgba(255, 255, 255, 5);
}

QPushButton:hover {
	background-color: rgba(255, 255, 255, 150);
}

QPushButton:pressed {
	background-color: rgba(255, 255, 255, 200);
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="spot_6">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	border: 1px solid black;
	background-color: rgba(255, 255, 255, 5);
}

QPushButton:hover {
	background-color: rgba(255, 255, 255, 150);
}

QPushButton:pressed {
	background-color: rgba(255, 255, 255, 200);
}</string>
           </property>
           <property name="text">
            <string>9</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="spot_7">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	border: 1px solid black;
	background-color: rgba(255, 255, 255, 5);
}

QPushButton:hover {
	background-color: rgba(255, 255, 255, 150);
}

QPushButton:pressed {
	background-color: rgba(255, 255, 255, 200);
}</string>
           </property>
           <property name="text">
            <string>8</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="spot_8">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	border: 1px solid black;
	background-color: rgba(255, 255, 255, 5);
}

QPushButton:hover {
	background-color: rgba(255, 255, 255, 150);
}

QPushButton:pressed {
	background-color: rgba(255, 255, 255, 200);
}</string>
           </property>
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="spot_9">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>45</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	border: 1px solid black;
	background-color: rgba(255, 255, 255, 5);
}

QPushButton:hover {
	background-color: rgba(255, 255, 255, 150);
}

QPushButton:pressed {
	background-color: rgba(255, 255, 255, 200);
}</string>
           </property>
           <property name="text">
            <string>7</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>25</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QPushButton" name="button_start">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-radius: 5px;
	color: white;
	background-color: rgb(23, 96, 255);
}

QPushButton:hover {
	background-color: rgba(23, 96, 255, 200);
}

QPushButton:pressed {
	background-color: rgba(23, 96, 255, 230);
}</string>
        </property>
        <property name="text">
         <string> Start</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>assets/start.png</normaloff>assets/start.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_shuffle">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-radius: 5px;
	color: white;
	background-color: rgb(23, 96, 255);
}

QPushButton:hover {
	background-color: rgba(23, 96, 255, 200);
}

QPushButton:pressed {
	background-color: rgba(23, 96, 255, 230);
}</string>
        </property>
        <property name="text">
         <string> Shuffle</string>
        </property>
        <property name="iconSize">
         <size>
          <width>15</width>
          <height>15</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_quit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	border-radius: 5px;
	color: white;
	background-color: rgb(202, 11, 11);
}

QPushButton:hover {
	background-color: rgba(202, 11, 11, 200);
}

QPushButton:pressed {
	background-color: rgba(202, 11, 11, 230);
}</string>
        </property>
        <property name="text">
         <string>Quit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
